### DISCUSSION TITLE
[iOs] Font-Problems persist

### DISCUSSION THREAD

**Discussion by TopperDEL** [unoplatform/uno#8953] [Q&A] [answered]
### Current behavior

I am still unable to add fonts to an iOs-Mobile-Project. Even the latest template does not correctly embedd the fluentui-font.

I've created a new project from the latest template and added a SymbolIcon to the Mainpage. It gets rendered as a Face on iOs, instead of a "+"-Symbol. Furthermore I've added a custom Font ("Pacifico-Regular.ttf") that does not get used on iOs.

I have a sample solution below.

### Expected behavior

Adding a font to the iOs-Resources should make the font available to iOs.

### How to reproduce it (as minimally and precisely as possible)

[TestFonts_iOs.zip](https://github.com/unoplatform/uno/files/8814430/TestFonts_iOs.zip)

### Workaround

None yet.

### Works on UWP/WinUI

**Comment by jeromelaban** [comment, marked as answer]
You can find additional information about [using fonts in uno here](https://platform.uno/docs/articles/features/custom-fonts.html?q=font).

In your sample, the Info.plist only contains the following:
```xml
	<key>UIAppFonts</key>
	<array>
	  <string>Fonts/uno-fluentui-assets.ttf</string>
	</array>
```
You will need to include your new font there as well.

**Reply by TopperDEL** [reply]
Ah, yeah - sorry, the sample does not have it. But my main project has. But still, the SymbolIcon renders incorrectly - so it seems the fluentui-asset is not loaded, too.

**Reply by jeromelaban** [reply]
I added the file to the plist, and this is what is showing up:
![image](https://user-images.githubusercontent.com/5839577/171437126-2c4e8aa7-723a-4b6a-8124-74241fd8d227.png)

Which version of Uno are you using in your project?

**Reply by TopperDEL** [reply]
That is really weird. I just built and deployed the app again with the added entry to the info.plist and it's working. In between I've only restarting both my win-pc and my mac (as I've paused work). Nothing else changed.

Consider this as resolved, thanks for your time. There seem to be little gremlins in my stack...

**Reply by TopperDEL** [reply]
My main problem shows the correct font now without any change. I've only rebuilt and redeployed it. Oh, what a day. :)
